content/design/pages/landing-page.md

```diff
@@ -0,0 +1,215 @@
+"""
+
+[[blocks]]
+layout = "links"
+title = "Links Block"
+copy = "Spotlight PA was founded to address a crisis in journalism in Pennsylvania: the ongoing and rapid decline of corporate-owned legacy news outlets, which is greatly diminishing access to trusted, contextual, and factual reporting across the state. With support from foundations, individual members, and more, Spotlight PA is uncovering stories that would otherwise go untold and cutting through partisanship and misinformation to deliver trusted, reliable journalism at no cost to all in Pennsylvania. Since its inception in 2019, Spotlight PA has become a leading national model for independent, collaborative journalism that empowers residents and gets results. Its work has won numerous state and national awards and driven meaningful change and policy reforms in the state capital and beyond."
+
+
+[[blocks.entries]]
+link = "/donate/"
+name = "How to Give to Spotlight PA"
+
+[[blocks.entries]]
+link = "/about/partners/"
+name = "Partners"
+
+[[blocks.entries]]
+link = "/about/faq/"
+name = "FAQ"
+
+[[blocks.entries]]
+link = "/about/statecollege/"
+name = "State College Regional Bureau"
+
+[[blocks]]
+layout = "series"
+title = "Series"
+series = ["Standard Series", "The PSERS Scandal"]
+[blocks.cta]
+url = "/investigations/"
+copy = "See More About Our Impact"
++++
+
+# Heading One (h1)
+
+Lorem ipsum dolor sit amet, *consectetur* adipisicing elit, sed do eiusmod
+tempor incididunt ut **labore et dolore magna aliqua**. Ut enim ad minim veniam,
+quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+consequat. ***Duis aute irure dolor*** in reprehenderit in voluptate velit esse
+cillum dolore eu fugiat nulla pariatur. ~~Excepteur sint occaecat~~ cupidatat non
+proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## Heading Two (h2)
+
+Lorem ipsum dolor sit amet, *consectetur* adipisicing elit, sed do eiusmod
+tempor incididunt ut **labore et dolore magna aliqua**. Ut enim ad minim veniam,
+quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+consequat. 
+
+---
+
+***Duis aute irure dolor*** in reprehenderit in voluptate velit esse
+cillum dolore eu fugiat nulla pariatur. ~~Excepteur sint occaecat~~ cupidatat non
+proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+### Third Level Heading (h3)
+
+unordered list:
+
+* item-1
+  * sub-item-1
+  * sub-item-2
+- item-2
+  - sub-item-3
+  - sub-item-4
++ item-3
+  + sub-item-5
+  + sub-item-6
+
+
+ordered list:
+
+1. item-1
+   1. sub-item-1
+   2. sub-item-2
+2. item-2
+   1. sub-item-3
+   2. sub-item-4
+3. item-3
+
+#### Heading Four (h4)
+
+##### Fifth Level Heading (h5)
+
+You may also want some images right in here like ![GitHub Logo](https://cloud.githubusercontent.com/assets/5456665/13322882/e74f6626-dc00-11e5-921d-f6d024a01eaa.png "GitHub") - you can do that but I would recommend you to use the component "image" and simply split your text.
```

Comment:
This image doesn't work. Maybe Github prohibits hot linking? I think just use an image on the Almanack file server instead.